pywr-schema/src/model.rs

```diff
@@ -1223,4 +1252,44 @@ mod core_tests {
 
         model.run::<ClpSolver>(&Default::default()).unwrap();
     }
+
+    #[test]
+    fn test_v1_conversion() {
+        let v1_str = include_str!("./test_models/timeseries_v1.json");
+        let v1: pywr_v1_schema::PywrModel = serde_json::from_str(v1_str).unwrap();
+
+        let (v2, errors) = PywrModel::from_v1(v1);
+
+        assert_eq!(errors.len(), 0);
+        assert_eq!(v2.network.nodes.len(), 6);
+        assert_eq!(v2.network.edges.len(), 5);
+
+        // Check that a timeseries ref has been created for a node attribute
```

Comment:
Would it be better/easier to have an expected v2 JSON file? Load up that schema and then compare the two for equality?